### PR TITLE
Fix css compilation issues that affect IE, particularly flexbox

### PR DESCRIPTION
**Pull request by jwforres**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1309522

### CODE REVIEW

**Comment by jwforres** [comment]
Thanks to @rhamilto for tracking down the issue

@spadgett PTAL  its a bit scary with the amount of diff its creating

**Comment by spadgett** [comment]
LGTM

**Comment by rhamilto** [comment]
Looks like it's fixed.

screen shot 2016-02-19 at 4 57 26 pm
screen shot 2016-02-19 at 4 33 53 pm
screen shot 2016-02-19 at 4 32 56 pm

**Comment by jwforres** [comment]
[merge]

**Comment by spadgett** [comment]
Bindata conflict :(

**Comment by jwforres** [comment]
[merge]

**Comment by spadgett** [comment]
Still doesn't like bindata.

```diff
Validating checked in bindata.go is up to date...
--- _output/test/assets/bindata.go	2016-02-22 12:24:46.697077252 -0500
+++ pkg/assets/bindata.go	2016-02-22 11:50:50.839796781 -0500
@@ -83075,7 +83075,11 @@
  * Copyright 2011-2015 Twitter, Inc.
  * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
  */
- /*! normalize.css v3.0.3 | MIT License | github.com/necolas/normalize.css */
+/*! normalize.css v3.0.3 | MIT License | github.com/necolas/normalize.css */
+hr,img{border:0}
+div.code,pre,textarea{overflow:auto}
make[1]: Leaving directory `/data/src/github.com/openshift/origin'
```

**Comment by jwforres** [comment]
hmm ok i'm going to do a full clean/install/build then

File: hack/install-assets.sh [hunk 1]
```diff
@@ -38,6 +38,10 @@ if ! which grunt > /dev/null 2>&1 ; then
 fi
 
 pushd ${OS_ROOT}/assets > /dev/null
+  # Check whether we need to force an update of the npm packages
+  if git diff master HEAD | grep 'assets/package.json' > /dev/null 2>&1 ; then
+    cmd "rm -rf node_modules"
```
Comment (jwforres) [inline hack/install-assets.sh]: @spadgett so the -force option just forces it to skip the npm cache, which didnt help this.  how do you feel about this change instead

**Comment by jwforres** [comment]
[test]

File: hack/install-assets.sh [hunk 2]
```diff
@@ -38,6 +38,10 @@ if ! which grunt > /dev/null 2>&1 ; then
 fi
 
 pushd ${OS_ROOT}/assets > /dev/null
+  # Check whether we need to force an update of the npm packages
+  if git diff master HEAD | grep 'assets/package.json' > /dev/null 2>&1 ; then
```
Comment (spadgett) [inline hack/install-assets.sh]: `git diff master HEAD --name-only` ?

File: hack/install-assets.sh (hunk 1 above)
Comment (spadgett) [inline hack/install-assets.sh]: OK with me.

**Comment by spadgett** [comment]
One small comment, otherwise LGTM.

File: hack/install-assets.sh (hunk 2 above)
Comment (jwforres) [inline hack/install-assets.sh]: good call, updated

**Comment by jwforres** [comment]
[merge]

**Comment by jwforres** [comment]
So we are past the package.json related issue, just looks like I need to
rebuild bindata to pick up other changes

**Comment by spadgett** [comment]
:(

```diff
--- _output/test/assets/bindata.go	2016-02-24 11:12:56.298316746 -0500
+++ pkg/assets/bindata.go	2016-02-24 10:11:05.202390222 -0500
@@ -86580,8 +86580,7 @@
 .table>tbody+tbody{border-top-width:1px}
 .table-borderless>tbody>tr>td,.table-borderless>tbody>tr>th,.table-borderless>tfoot>tr>td,.table-borderless>tfoot>tr>th,.table-borderless>thead>tr>td{border-top:none}
 .table .pull-spec{max-width:700px}
-@media (max-width:1199px){.table .pull-spec{max-width:315px}
-}
+@media (max-width:1199px){.table .pull-spec{max-width:315px}}
 @media (max-width:767px){.table-mobile{border-top:2px solid #d1d1d1}
```

**Comment by jwforres** [comment]
yeah i rebased and pushed before i tried rebuilding bindata so i was expecting those to fail, the commit that its running now has the fix

**Pull request merged**